formats/protobuf/src/main/java/io/cloudevents/protobuf/ProtoDataWrapper.java

```diff
@@ -35,4 +38,49 @@ public Message getMessage() {
     public byte[] toBytes() {
         return protoMessage.toByteArray();
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
why do we need the equal method? It it is for testing only I wouldn't add it here

Reply:
There is a an existing 'equals' implementation for the CloudEvent object and the current 'core' wrapper implementations have implemented the equality method as well so to be able to cleanly compare CloudEvent objects the various implementations really should be doing this as well - granted it's probably more useful during UnitTesting but externalizing all those types of comparisons would be a major undertaking I think .. all other CloudEvent roundTrip tests I've written have relied on that equality test to act as proof.

See: https://github.com/cloudevents/sdk-java/blob/main/core/src/main/java/io/cloudevents/core/data/BytesCloudEventData.java#L30